javaparser-testing/src/test/java/com/github/javaparser/JavaParserTest.java

```diff
@@ -207,6 +207,6 @@ public void trailingCodeIsAnError() {
     @Test
     public void trailingWhitespaceIsIgnored() {
         BlockStmt blockStmt = JavaParser.parseBlock("{} // hello");
-        assertEquals("\"}\"   <94>   (line 1,col 2)-(line 1,col 2)", blockStmt.getTokenRange().get().getEnd().toString());
+        assertEquals("\"}\"   <95>   (line 1,col 2)-(line 1,col 2)", blockStmt.getTokenRange().get().getEnd().toString());
```

Comment:
what this "95" represents here?

Reply:
It the token type. Annoying, but it did remind me to put the new token type in the big category switch.

Reply:
Maybe we can add a comment about that